infra/app/app-config/env-config/outputs.tf

```
@@ -1,10 +1,11 @@
 output "database_config" {
   value = var.has_database ? {
-    cluster_name       = "${var.app_name}-${var.environment}"
-    access_policy_name = "${var.app_name}-${var.environment}-db-access"
-    app_username       = "app"
-    migrator_username  = "migrator"
-    schema_name        = var.app_name
+    cluster_name               = "${var.app_name}-${var.environment}"
+    app_access_policy_name     = "${var.app_name}-${var.environment}-app-access"
+    migrate_access_policy_name = "${var.app_name}-${var.environment}-migrate-access"
```

nit: can we rename to migrator_access_policy_name
```suggestion
    migrator_access_policy_name = "${var.app_name}-${var.environment}-migrate-access"
```

---

Replaced in separate commit including other instances and references